geode-core/src/main/java/org/apache/geode/cache/Region.java

```diff
@@ -1359,10 +1359,30 @@ Object selectValue(String queryPredicate) throws FunctionDomainException, TypeMi
    * in the specified map. This operation will be distributed to other caches if the scope is not
    * <code>Scope.LOCAL</code>.
    *
-   * @param map the key/value pairs to put in this region.
-   * @since GemFire 5.0
+   * If any exception is thrown due to this call, it can imply that there may have been a partial
+   * update performed on the region. Use putAll from within a transaction to get atomicity with all
+   * the
```

Comment:
"the entries" are they in two different lines or it is my browser. If they are two lines, it will be nice to have in the same line.

Reply:
fixed.